tests/template_backends/test_dummy.py

```diff
@@ -55,6 +56,25 @@ def test_html_escaping(self):
         self.assertIn('&lt;script&gt;', content)
         self.assertNotIn('<script>', content)
 
+    def test_django_html_escaping(self):
+        if self.backend_name == 'dummy':
+            return
+
+        class TestForm(Form):
+            test_field = CharField()
+
+        media = Media(js=['my-script.js'])
+
+        template = self.engine.get_template('template_backends/django_escaping.html')
+        content = template.render({'media': media, 'test_form': TestForm()})
+
+        self.assertIn('<script', content)
```

Comment:
It doesn't seem the output should be too lengthy, could we use `assertHTMLEqual()` instead?